apps/server/package.json

```diff
@@ -49,6 +49,7 @@
     "dev:test": "cross-env IS_TEST=true nodemon --exec \"ts-node-esm\" ./src/index.ts",
     "prebuild": "pnpm setdb",
     "build": "pnpm prebuild && esbuild src/app.ts --log-level=error --platform=node --format=cjs --bundle --minify --outfile=dist/index.cjs",
+    "build:electron": "pnpm prebuild && esbuild src/app.ts --log-level=error --platform=node --format=cjs --bundle --minify --outfile=dist/index.cjs",
```

Comment:
this is the "big" difference
building `src/app.ts` and not `src/index.ts` 

I made a separate electron script to simplify the build to one turbo command but maybe there is a better way?

Reply:
oh I see, but then do we still need the `build:local` or is just a broken script? could the new script replace `build:local` instead?

Reply:
if one wants to to run the apps/sever an apps/electron separate then  `local`  is the one that is needed